rpm/bluez5.spec

```diff
@@ -293,6 +293,7 @@ systemctl-user daemon-reload ||:
 %{_bindir}/hciconfig
 %{_bindir}/hcitool
 %{_bindir}/hex2hcd
+%{_bindir}/isotest
```

Comment:
Just a minor remark : the version is still at 5.63 in the spec file. (I know that OBS is overriding this, but well).

Reply:
Fixed